scripts/app.js

```diff
@@ -9,20 +7,34 @@ function Project (opts) {
   this.body = opts.body;
 }
 
+Project.all = [];
+
 Project.prototype.toHtml = function() {
   var source = $('#project-template').html();
   var template = Handlebars.compile(source);
   return template(this);
 };
 
-projectData.sort(function(a,b) {
-  return (new Date(b.publishedOn)) - (new Date(a.publishedOn));
-});
+Project.loadAll = function(rawData) {
+  rawData.sort(function(a,b) {
```

Comment:
You cant sort rawData here like this because .sort() only works on an array.  At this point rawData is a string so you will need to parse it.  (This is a hint)